### PR TITLE
Add STM32F0 Discovery board with basic examples.

### PR DESCRIPTION
**Pull request by strongly-typed**
In progress of debugging USB2CAN with STM32F0.

### CODE REVIEW

**Review by salkinium — DISMISSED** [review]
LGTM so far!

**Comment by salkinium** [comment]
Don't forget to add it to Travis too, job 2, 3 or 4.

**Comment by strongly-typed** [comment]
Thanks for the hint! Just added it.

**Review by salkinium — APPROVED** [review]
🚢🕴

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ Here is a list of supported **and tested** microcontrollers and development boar
 | ATtiny85   | custom                  | &#9733;&#9733;&#9733;               |
 | ATmega328p | [Arduino Uno][]         | &#9733;&#9733;&#9733;               |
 | STM32F031  | [Nucleo F031K6][]       | &#9733;&#9733;&#9733;&#9733;        |
+| STM32F051  | [STM32F0 Discovery][]   | &#9733;&#9733;                      |
```
Comment (salkinium) [inline README.md:65]: and this is missing a link ;-)

**Pull request merged**